rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -531,6 +550,42 @@ impl From<EvmMultiRpcResult<EvmBlock>> for ReducedResult<Block> {
     }
 }
 
+impl From<EvmMultiRpcResult<Vec<EvmLogEntry>>> for ReducedResult<Vec<LogEntry>> {
+    fn from(value: EvmMultiRpcResult<Vec<EvmLogEntry>>) -> Self {
+        fn map_logs(logs: Vec<EvmLogEntry>) -> Result<Vec<LogEntry>, String> {
+            logs.into_iter().map(map_single_log).collect()
+        }
+
+        fn map_single_log(log: EvmLogEntry) -> Result<LogEntry, String> {
+            Ok::<LogEntry, String>(LogEntry {
```

Comment:
Why not simply `Ok(Log Entry {`?

Reply:
you're right, it's not necessary here and can be simplified as you suggested. Done.